docs/rules/images-no-direct-imports.md

```diff
@@ -0,0 +1,30 @@
+# Prevent directly importing image files and instead force usage of an index file
+
+Image files should live in a directory (e.g. `icons`, `illustrations` or `images`) that must contain a dedicated index file that reexports all the images in that folder.
```

Comment:
How do you feel about this instead for these 2 paragraphs?

>This rule prevents direct imports to image files by enforcing that all images be placed inside a directory (e.g. `icons`, `illustrations` or `images`) that also contains a dedicated index file that re-exports from that folder. Files that consume images must import from that folder instead of importing from the path to the image file directly.

Reply:
I think that implies something slightly different to what is happening. To me that paragraph suggests that is the images that are checked by eslint (e.g. "enforcing all images..."), which is not the case.

The implementation of this rule it don't check anything about the images themselves, it identifies imports to those images and complains if the import is in any file except the

Reply:
How about the following:

> Image files should live in a directory (e.g. `icons`, `illustrations` or `images`) that must contain a dedicated index file that re-exports all the images in that folder.
>
> Files that consume images must import from that index file instead of importing images directly. This rule enforces that convention by disallowing importing directly from an image in any file except the index file in the same folder as the image.

Reply:
That sounds good to me 👍